pulpcore/app/apps.py

```diff
@@ -273,6 +284,14 @@ def _populate_system_id(sender, apps, verbosity, **kwargs):
         SystemID().save()
 
 
+def _ensure_default_domain(sender, apps, verbosity, **kwargs):
```

Comment:
```suggestion
def _ensure_default_domain(sender, **kwargs):
```
... unless you want to add some verbosity...